pysilcam/silcam_classify.py

```diff
@@ -35,13 +29,13 @@ def check_model(model_path):
         raise Exception(header_file + ' not found')
```

Comment:
In check_model, add a check for the correct (updated) keras model, and raise an error with a description that this is an update that requires re-download of the keras model from the github wiki. See section on 'Setup pysilcam' here: https://github.com/SINTEF/PySilCam/wiki/Installation,-setup-and-contributions

Reply:
Perhaps we should use more specific and informative exceptions here, e.g. FileNotFoundError for the os.path.exists check, and TypeError or ValueError for the is_hdf5 part.

Also, instead of "if os.path.exists(path) is False", I suggest the more Pythonic "if not os.path.exists(path)".